src/inert.js

```diff
@@ -72,10 +73,10 @@ class InertRoot {
     // Watch for:
     // - any additions in the subtree: make them unfocusable too
     // - any removals from the subtree: remove them from this inert root's managed nodes
-    // - attribute changes: if `tabindex` is added, or removed from an intrinsically focusable element,
-    //   make that node a managed node.
+    // - attribute changes: if `tabindex` is added, or removed from an intrinsically focusable
+    //   element, make that node a managed node.
     this._observer = new MutationObserver(this._onMutation.bind(this));
-    this._observer.observe(this._rootElement, { attributes: true, childList: true, subtree: true });
```

Comment:
Why remove these spaces? Does ESLint insist on it? Is there an option to make it not do that?